### PR TITLE
feat: add a new plugin multion

### PR DESCRIPTION
**Pull request by LinkW77**
# Pull Request

## PR Description

Add a new plugin multion

## Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Performance enhancement
- [ ] Code refactor
- [ ] Documentation update
- [ ] Other, please describe:

## Checklist

Before submitting this PR, please make sure:

- [x] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) guidelines.
- [x] I have tested my changes locally to ensure they are effective.
- [x] I have updated the necessary documentation (if applicable).

### CODE REVIEW

**Review by DynamesC — APPROVED** [review]
LGTM

File: plugin/bundles/multion/plugins/webpage_retrieve/plugin.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    async def execute(self, credentials: BundleCredentials, plugin_input: PluginInput) -> PluginOutput:
+        cmd: str = plugin_input.input_params.get("cmd")
+        url: str = plugin_input.input_params.get("url", None)
+        fields: List[str] = plugin_input.input_params.get("fields", None)
+        multion_api_key: str = credentials.credentials.get("MULTION_API_KEY")
+
+        if url and fields:
+            api_url = f"https://api.multion.ai/v1/web/retrieve"
+            data = {"cmd": cmd, "url": url, "fields": fields}
+        else:
```
Comment (jameszyao) [inline plugin/bundles/multion/plugins/webpage_retrieve/plugin.py:21]: Split into two plugins

**Review by jameszyao — APPROVED** [review]
LGTM

**Review by jameszyao — APPROVED** [review]
LGTM

**Review by DynamesC — APPROVED** [review]
LGTM

**Pull request merged**